### ISSUE TITLE
Multi-class support

### ISSUE THREAD

**Issue by jeremiedb** [Evovest/EvoTrees.jl#13]
Add softmax loss function

**Comment by jeremiedb** [comment]
done
